webui/react/src/hooks/useAuthCheck.ts

```diff
@@ -8,7 +8,7 @@ import { routeAll } from 'routes/utils';
 import { updateDetApi } from 'services/apiConfig';
 import { AUTH_COOKIE_KEY, setAuth, setAuthChecked } from 'stores/auth';
 import { initInfo, useDeterminedInfo } from 'stores/determinedInfo';
-import { getCookie } from 'utils/browser';
+import { getCookie } from 'utils/cookie';
```

Comment:
**nit question:** curious what led you to split off `utils/cookie`

Reply:
I had the same issue as using the observable
```
Uncaught ReferenceError: Cannot access 'serverAddress' before initialization
    at updatedApiConfigParams (apiConfig.ts:19:15)
    at generateApiConfig (apiConfig.ts:25:18)
    at apiConfig.ts:53:21
```